jsonschema/validators.py

```diff
@@ -179,6 +179,14 @@ def create(
             will be converted to functions and redefined in this object's
             `jsonschema.TypeChecker`.
 
+        id_of:
+        
+            Function that gets $id from a schema.
```

Comment:
Will leave a more specific comment but the point of this function is to essentially abstract over `$id` (newer drafts) vs `id` (older drafts) vs whatever crazy new way we come up with for future drafts, so I'd rather not mention a specific name of a validator, would be a bit better to just say what the id *concept* is, which is return the ID of the schema (so, this is a function that, given a schema, returns its ID).

Make sense?